apps/prairielearn/src/tests/groupExam.test.js

```diff
@@ -0,0 +1,401 @@
+const { config } = require('../lib/config');
+const assert = require('chai').assert;
+const cheerio = require('cheerio');
+const fetch = require('node-fetch').default;
+const { step } = require('mocha-steps');
+
+const sqldb = require('@prairielearn/postgres');
+const sql = sqldb.loadSqlEquiv(__filename);
+
+const helperServer = require('./helperServer');
+const { idsEqual } = require('../lib/id');
+const { TEST_COURSE_PATH } = require('../lib/paths');
+
+let page, elemList;
```

Comment:
Are these actually shared between any tests? If not, just make them local variables; it's easier to reason about.